scripts/import_dvv.sql

```diff
@@ -0,0 +1,38 @@
+-- Insert buildings to jkr_rakennus
+insert into jkr.rakennus (prt, kiinteistotunnus, onko_viemari, geom, kayttoonotto_pvm, rakennuksenkayttotarkoitus_koodi, rakennuksenolotila_koodi)
+select 
+    rakennustunnus as prt,
+    "sijaintikiinteistön tunnus" as kiinteistotunnus,
+    viemäri::boolean as onko_viemari,
+    ST_GeomFromText('POINT('||itä_koordinaatti||' '||pohjois_koordinaatti||')', 3067) as geom,
+    to_date("käytössä_olotilanteen muutospäivä"::text, 'YYYYMMDD') as kayttoonotto_pvm,
```

Comment:
Fixed. This caused the ugliest sql I've ever written tho, because
1) the name of the field, for some unfathomable reason, is "valmis_tumis_
_päivä" including the newline, and
2) the field contains errors which require checking the string before converting it to date.